src/dashboard/form-manager/components/FieldRenderer.tsx

```
@@ -0,0 +1,152 @@
+            mappedField.type = field.id.endsWith( '_ids' ) ? 'array' : 'text';
+            const elements = Array.isArray( field.options )
+                ? field.options.map( ( opt: any ) => ( {
+                      label: opt.label,
+                      value: opt.value,
+                  } ) )
+                : Object.entries( field.options ).map(
+                      ( [ value, label ] ) => ( {
+                          label,
+                          value,
+                      } )
+                  );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)